src/mavlink-router/main.cpp

```diff
@@ -660,11 +677,15 @@ static int parse_confs(ConfFile &conf)
         char *addr;
         bool eavesdropping;
         unsigned long port;
+        char *filter_inc;
```

Comment:
Can you also add them for TCP and UART for consistency?

Reply:
Added.